integration-tests/integration_tests/test_scenarios/int_async_express_tests.py

```diff
@@ -0,0 +1,70 @@
+from unittest import TestCase
+
+from integration_tests.helpers import methods, message_retriever
+
+
+class FunctionalTest(TestCase):
+
+    def test_async_express_outbound_status(self):
+        # the response is just an acknowledgement from spine...
+        outbound_response, _, _ = methods.get_interaction_from_template('async express',
+                                                                        'QUPC_IN160101UK05',
+                                                                        '9689177621',
+                                                                        'Asynchronous Express test')
+        # we need to 'accept' the message in the queue, so it is removed and does impact on subsequent tests
+        message_retriever.get_inbound_response()
+        self.assertTrue(methods.check_status_code(outbound_response, 202),
+                        "Async Express outbound test failed")
+
+    def tests_assert_ssl_with_proton(self):
+        import proton
+        self.assertTrue(proton.SSL.present())
```

Comment:
Just remembered this; I forgot to delete this test that I was using to try and diagnose the problem getting this to build in jenkins, feel free to delete this